crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -436,12 +436,28 @@ pub struct FluidData {
 
 pub const FLUID_DATA_DESCRIPTOR: &str = "RklEPUNPTU1PTi5BUFBMRS5JTkFQUC5QQVlNRU5U";
 
+pub const FLUID_DATA_DESCRIPTOR_FOR_SAMSUNG_PAY: &str =
+    "RklEPUNPTU1PTi5TQU1TVU5HLklOQVBQLlBBWU1FTlQ=";
```

Comment:
Rather, keep the plaintext value `FID=COMMON.SAMSUNG.INAPP.PAYMENT` here in the constant, then base64 encode it in code.